src/download/exchangeDownloader.ts

```diff
@@ -259,15 +256,40 @@ export async function getVersionByDeployment(
     return;
   }
 
-  if (!asset.version) {
+  if (!asset.versionGroups) {
     ramlToolLogger.error(
-      `${logPrefix} The rest API ${restApi.assetId} is missing the asset.version`
+      `${logPrefix} The rest API ${restApi.assetId} is missing asset.versionGroups`
     );
     return;
   }
+  const versions: string[] = [];
+  asset.versionGroups.forEach((versionGroup) => {
+    const version = getLatestReleaseVersion(versionGroup);
+    if (version) {
+      versions.push(version);
+    }
+  });
+  return versions;
+}
 
-  // return the most recent version of an asset from the rest API
-  return asset.version;
+function getLatestReleaseVersion(versionGroup: {
+  versions: Array<{ version: string }>;
+}): void | string {
+  if (!versionGroup.versions || versionGroup.versions.length === 0) {
+    return;
+  }
+  const releaseAssetVersions = versionGroup.versions.filter((version) => {
+    return releaseSemverRegex.test(version.version);
+  });
+  // Sort versions and get the latest
+  return releaseAssetVersions.sort((instanceA, instanceB) => {
```

Comment:
This function is a nice solution, it feels like a very clean way to get the latest version, straightforward to understand